railties/lib/rails/test_unit/minitest_plugin.rb

```diff
@@ -60,7 +60,7 @@ def self.plugin_rails_options(opts, options)
   # Running several Rake tasks in a single command would trip up the runner,
   # as the patterns would also contain the other Rake tasks.
   def self.rake_run(patterns) # :nodoc:
-    @rake_patterns = patterns
+    (@rake_patterns ||= []).push(*patterns)
```

Comment:
Won't this negate what the comment says above? In other words, do we need `rake_run` anymore? It might have become obsolete after moving to `autorun`.